### DISCUSSION TITLE
$timeout in Job doesn't work

### DISCUSSION THREAD

**Discussion by handhikadj** [laravel/framework#45078] [Q&A] [answered]
- Laravel Version: 9.41.0
- PHP Version: 8.1.9
- Database Driver & Version: MySQL 8

### Description:
```php
class MyQueue implements ShouldQueue
{
    use Dispatchable, InteractsWithQueue, Queueable, SerializesModels;

    public $timeout = 2;

    /**
     * Create a new job instance.
     *
     * @return void
     */
    public function __construct()
    {
      //
    }

    /**
     * Execute the job.
     *
     * @return void
     */
    public function handle()
    {
        while (true) {}
    }
}
```
I set the timeout to be 2 secs, can't get timed out. 
pcntl enabled
<img width="151" alt="image" src="https://user-images.githubusercontent.com/22522638/203838216-d7836948-2120-4135-a403-11f846d6415b.png">

### Steps To Reproduce:

create a job like above code. execute it and monitor it with `queue:work/listen`

**Comment by Kamandlou** [comment]
Use this command:
`php artisan queue:listen --timeout=2`

**Reply by handhikadj** [reply]
This time, I want to set it on the class

**Comment by handhikadj** [comment, marked as answer]
problem solved. I didn't notice that `pcntl` was not installed on supervisor service on docker compose
